policies/README.md

```diff
@@ -30,3 +30,15 @@ psql -U postgres -c "select * from aws_policy_results" --csv
 # HTML output
 psql -U postgres -c "select * from aws_policy_results" --html
 ```
+
+## Visualization
+
+Currently we have a pre-built compliance dashboard on top of `aws_policy_results` table which is available [here](../dashboards/grafana/compliance.json)
+
+<img alt="Azure Asset Inventory Grafana Dashboard" src="../dashboards/grafana/compliance.png" width=50% height=50%>
+
+### Instaltion
+
+1. Execute one or more of the above policies.
+2. Add the CloudQuery postgres database as a data source to Grafana (`Configuration -> Data Sources -> Add Data Source`)
+3. Import [dashboards/azure/azure_asset_inventory.json](../dashboards/grafana/compliance.json) into Grafana (`Import -> Upload JSON File`).
```

Comment:
@yevgenypats I think we should move most of this `Visualization` section to a separate readme under the `dashboards` directory, and link to that section from here (maybe while keeping the screenshot). That way, you can look at the `dashboards` directory on its own and still know how to use it. WDYT?

Reply:
Yeah possible. I would still link to Visualization then from this README and also backlink from visualizations to policies.

Reply:
Yup, that's what I had in mind with my comment too 👍